posts/2020/03/Spyder-4.md

```diff
@@ -0,0 +1,31 @@
+
+Spyder 4 is the first version developed with financial support from several companies as well as donations by the international user community. But, as a project, we couldn't have been able to reach the level of maturity needed to receive and handle that support, without the pivotal opportunities Travis Oliphant, former CEO of Anaconda and current one of Quansight, gave me to work in Scientific Python. Thanks to him, I became part of Anaconda, Inc. in 2015, I was able to hire a small small team to improve Spyder within Anaconda in late 2016, and I entered Quansight to work solely on Spyder in 2018. As with other projects in our ecosystem, such as [Bokeh](https://github.com/bokeh/bokeh), [Dask](https://github.com/dask/dask) and [Numba](https://github.com/numba/numba), Spyder benefited immensely from Travis' trust in the role these projects and ours could play in the future. He certainly believed in the vision their maintainers worked so hard to make a reality of, even if their beginnings were humble and their chances of survival uncertain. Therefore, my first big acknowledgment is to Travis: thanks for giving us a chance!
+
+I also want to thank our community for its continued support. As I've witnessed during my years as Spyder's lead developer, many newbies and veterans alike keep choosing Spyder as a viable alternative to do scientific programming in Python. It's really you, the members of this fantastic community, which maintain Spyder relevant in a highly competed field by using and contributing back to it. I'm honored and humbled to have received such first-class additions to Spyder 4 from our volunteer contributors as our new Plots pane, thanks to [Jean-Sébastien Gosselin](https://github.com/jnsebgosselin); a complete re-architecturing of our IPython console, which allowed a lot of improvements to our debugger, thanks to [Quentin Peter](https://github.com/impact27); auto-saving functionality for our editor, thanks to [Jitse Niesen](https://github.com/jitseniesen); the initial implementation of the `runcell` command, thanks to [Brian Olsen](https://github.com/bcolsen); and significant improvements to our documentation, thanks to [CAM Gerlach](https://github.com/CAM-Gerlach).
```

Comment:
```suggestion
I also want to thank our community for its continued support. As I've witnessed during my years as Spyder's lead developer, many newbies and veterans alike keep choosing Spyder as their primary tool for scientific programming in Python. It's really you, the members of this fantastic community, which keep Spyder relevant in a highly competitive field by using and contributing back to it. I'm honored and humbled to have received such first-class additions to Spyder 4 from our volunteer contributors as our new Plots pane, thanks to [Jean-Sébastien Gosselin](https://github.com/jnsebgosselin); a complete re-archetecting of our IPython console, which enabled numerous improvements to our debugger, thanks to [Quentin Peter](https://github.com/impact27); auto-saving functionality for our editor, thanks to [Jitse Niesen](https://github.com/jitseniesen); the initial implementation of the `runcell` command, thanks to [Brian Olsen](https://github.com/bcolsen); and significant improvements to our documentation, thanks to [CAM Gerlach](https://github.com/CAM-Gerlach).
```

Technically speaking, Jean-Sébastien, Jitse, Brian and I were all on the core dev team during our primary time working on Spyder 4, not just community contributors, and the improvements to the docs I made were for Spyder 3, not 4. I don't feel its out of place for me (if I belong here at all) since I didn't contribute to Spyder 4 much at all, the vast majority of my work was for Spyder 3, the Spyder 3 docs, the Spyder 3 issue tracker, all that other stuff. But I would be concerned that mentioning them here could come across as putting them in a lesser, lower category compared to the other core devs mentioned below when, to my understanding, that wasn't the case at the time.

Reply:
> I don't feel its out of place for me (if I belong here at all) since I didn't contribute to Spyder 4 much at all, the vast majority of my work was for Spyder 3, the Spyder 3 docs, the Spyder 3 issue tracker, all that other stuff.

Ok, I'll leave here then.

> But I would be concerned that mentioning them here could come across as putting them in a lesser, lower category compared to the other core devs mentioned below when, to my understanding, that wasn't the case at the time.

I agree with this. I'll merge this paragraph with the one below about the Colombian team.